.gitignore

```diff
@@ -16,3 +16,6 @@
 /test/src/*.dylib
 /test/src/*.o
 /test/src/*.so
+
+# Do NOT ignore binaries placed in subdirs under '/test/bin'.
+!/test/bin/*/*
```

Comment:
I suspected something like that. No worries, it's no big deal! It just caught my attention. :wink: